tasks.rb

```diff
@@ -2174,7 +2175,9 @@ def set_measure_argument_values(hpxml_file, args, sch_args)
   end
 
   # Misc
-  if ['base-misc-defaults.xml'].include? hpxml_file
+  if ['base-misc-additional-properties.xml'].include? hpxml_file
+    args['additional_properties'] = 'foo=1|bar|description=test|comma=,|special=<|special2=>|special3=/|special4="\"'
```

Comment:
I assume you wanted `<special4>\<special4>`, but this code produces `<special4>'\'<special4>`.